coops-ui/tests/unit/Dashboard.spec.ts

```diff
@@ -55,19 +55,19 @@ describe('Dashboard - UI', () => {
 
     expect(wrapper.vm.hasBlockerFiling).toEqual(false)
     expect(wrapper.vm.$el.querySelector('#btn-standalone-addresses')
-      .getAttribute('disabled')).toBeFalsy()
+      .getAttribute('disabled')).not.toBe('true')
     expect(wrapper.vm.$el.querySelector('#btn-standalone-directors')
-      .getAttribute('disabled')).toBeFalsy()
+      .getAttribute('disabled')).not.toBe('true')
   })
 
   it('disables standalone filing buttons when there is a blocker filing in the to-do list', () => {
     wrapper.find(TodoList).vm.$emit('has-blocker-filing', true)
 
     expect(wrapper.vm.hasBlockerFiling).toEqual(true)
     expect(wrapper.vm.$el.querySelector('#btn-standalone-addresses')
-      .getAttribute('disabled')).toBeTruthy()
+      .getAttribute('disabled')).toEqual('true')
```

Comment:
Does this work? When this attribute is set, I thought its value is actually "disabled" (ie, `disabled="disabled"`)?

Reply:
It works. It returns 'true' as the value of this attribute is a boolean.  `hasBlockerFiling`

Reply:
OK, thanks.